1e/metadata.csv

```diff
@@ -0,0 +1,17 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+experience.score,gauge,,score,,"A score that provides a weighted indicator of device performance, stability, responsiveness and user sentiment",0,1e,experience score,
```

Comment:
The metrics need a prefix that conforms to https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/#rules-and-best-practices-for-naming-metrics

Reply:
Added onee as prefix